### DISCUSSION TITLE
Which entities to use

### DISCUSSION THREAD

**Discussion by cherowley** [BottlecapDave/HomeAssistant-OctopusEnergy#1039] [General]
Hi!

Been using this for many months now. Did an update around the 8th October and since then the cumulative gas/electric no longer update. Also the current elec usage has frozen at a fixed value..

What are the correct entities to use for these 3 values? Other values such as price etc still continue to work..

Thanks!

**Comment by BottlecapDave** [comment]
The sensors update whenever new data is available via the OE site. My first question is do you have data available on the OE website past 8th? If you do, then I suggest updating to the latest available version (v13.0.2) as how the sensors update has been changed (it used to always be a set number of days behind, but now it retrieves the latest available days worth of data).

**Reply by cherowley** [reply]
Many thanks BC. Embarrassingly, I unplugged the little pink octopus mini to reboot it and it seems to be sending all the entities again now?!

I rather stupidly assumed that as a good portion of the entities were still coming across fine, that it couldn't be anything to do with the pink home mini thing :)
